go.mod

```diff
@@ -17,6 +17,7 @@ require (
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.5.1
 	go.uber.org/zap v1.10.0
+	golang.org/x/mod v0.2.0
```

Comment:
Please update the `ATTRIBUTION.md` file to include the BSD3 license attribution for this new dependency:

https://pkg.go.dev/golang.org/x/mod/modfile?tab=licenses

Reply:
Nice catch! Done.